scripts/package.validate.mjs

```diff
@@ -103,160 +85,131 @@ for (const workspacePath of workspacePaths) {
-  const modulePath = path.join(workspacePath, packageJson.module);
+  const modulePath = path.join(workspacePath, packageJson.module)
   if (!fs.existsSync(modulePath)) {
-    console.error(`Missing ${modulePath}`);
-    process.exit(1);
+    console.error(`Missing ${modulePath}`)
+    process.exit(1)
   }
 
   // assert "types" field is "dist/types/src/index.d.ts"
   if (packageJson.types !== "dist/types/src/index.d.ts") {
-    console.error(
-      `Invalid ${packageJsonPath} types field: ${packageJson.types}`
-    );
-    process.exit(1);
+    console.error(`Invalid ${packageJsonPath} types field: ${packageJson.types}`)
+    process.exit(1)
   }
 
   // assert "types" field points to a real file
-  const typesPath = path.join(workspacePath, packageJson.types);
+  const typesPath = path.join(workspacePath, packageJson.types)
   if (!fs.existsSync(typesPath)) {
-    console.error(`Missing ${typesPath}`);
-    process.exit(1);
+    console.error(`Missing ${typesPath}`)
+    process.exit(1)
   }
 
   // assert "react-native" field is "src/index.ts"
   if (packageJson["react-native"] !== "src/index.ts") {
-    console.error(
-      `Invalid ${packageJsonPath} react-native field: ${packageJson["react-native"]}`
-    );
-    process.exit(1);
+    console.error(`Invalid ${packageJsonPath} react-native field: ${packageJson["react-native"]}`)
+    process.exit(1)
   }
 
   // assert "react-native" field points to a real file
-  const reactNativePath = path.join(workspacePath, packageJson["react-native"]);
+  const reactNativePath = path.join(workspacePath, packageJson["react-native"])
   if (!fs.existsSync(reactNativePath)) {
-    console.error(`Missing ${reactNativePath}`);
-    process.exit(1);
+    console.error(`Missing ${reactNativePath}`)
+    process.exit(1)
   }
 
-  // assert "exports" field is an object with "default", "import", "react-native", and "types" fields
+  // assert "exports" field is an object with "default", "import", and "types" fields
   if (
     !packageJson.exports ||
     typeof packageJson.exports !== "object" ||
     Array.isArray(packageJson.exports) ||
     !packageJson.exports.default ||
     !packageJson.exports.import ||
-    !packageJson.exports["react-native"] ||
     !packageJson.exports.types
   ) {
     console.error(
-      `Invalid ${packageJsonPath} exports field: ${JSON.stringify(
-        packageJson.exports,
-        null,
-        2
-      )}`
-    );
-    process.exit(1);
+      `Invalid ${packageJsonPath} exports field: ${JSON.stringify(packageJson.exports, null, 2)}`
+    )
+    process.exit(1)
+  }
+
+  // assert "exports.react-native" field does not exist
+  if (packageJson.exports["react-native"]) {
+    console.error(
+      [
+        `Invalid ${packageJsonPath} exports.react-native field should not exist`,
+        `Remove this check once typescript types are more stable`,
+        `See https://github.com/infinitered/reactotron/issues/1430`,
+      ].join("\n")
+    )
+    process.exit(1)
```

Comment:
Prettier formatting made a bunch of noise in this file but this is the actual change